src/AutoRest.CSharp.V3/Output/Models/Requests/RequestParameter.cs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using AutoRest.CSharp.V3.Output.Models.Shared;
+using AutoRest.CSharp.V3.Output.Models.TypeReferences;
+
+namespace AutoRest.CSharp.V3.Output.Models.Requests
+{
+    internal struct RequestParameter
```

Comment:
I think this name is less descriptive than before.

Reply:
Added descriptiveness